### DISCUSSION TITLE
Are you using this library?

### DISCUSSION THREAD

**Discussion by Bindambc** [Bindambc/whatsapp-business-java-api#26] [Q&A]
Are you using the library? Talk about your experience.

**Comment by shingisikula** [comment]
Hello. I've included your SDK in my project. 
I'm trying to log the API calls when I send messages using the graph api

`var message = getMessage(fromNumber);
MessageResponse messageResponse = whatsappBusinessCloudApi.sendMessage(phoneNumberId, message);`

can you provide a way to specify a custom Interceptor to be used in `WhatsappApiServiceGenerator.createService`?

**Reply by Bindambc** [reply]
Hello @shingisikula, what information would you like to display in the log? request body? headers ?

**Reply by shingisikula** [reply]
I'm looking to log out the request body
@Bindambc

**Comment by jhsiqueira13** [comment]
Hi, ai need create template with image and/or Video header.
I do the upload of the image/video and receive a ID
But, when i will create a template, requires a stranger(for me) id

                        .setExample(new Example().addHeaderHandleExamples("4::aW1hZ2UvanBlZw==:ARZdpGlLrA9uwIGGZc-UFu5viAD1BkqTCYGL8je2d7xovDZphaWG8gJPTSJfekNchsL3SWdY8-jTA9ZRq_MWro-1wfJnApfbb0ByrUoDb6nNZA:e:1679805732:3449824985304722:100002914375136:ARauotmqlFdTcNENzt0"))

How i get this "hash" handler example?

ps: congratulations for the project, its amazing.
